json-guides/bulkhead.json

```diff
@@ -122,9 +123,11 @@
             "name": "AddingBulkheadAnnotation",
             "title": "Adding the @Bulkhead annotation",
             "description": [ 
-                "Semaphore style Bulkhead"         
+                "Now that you've seen how our application can easily fail when overloaded, we will fix that by using the Bulkhead pattern. Add the @Bulkhead annotation to the requestDoctor microservice to limit the maximum concurrent requests.<br>",
```

Comment:
Instead of requestDoctor microservice, it is requestAVirtualFinancialAdvisor microservice.

Reply:
Fixed